package.json

```diff
@@ -151,6 +151,7 @@
     "jsdom": "^13.0.0",
     "lint-staged": "^8.2.1",
     "mkdirp": "^0.5.1",
+    "mocha-junit-reporter": "^2.0.0",
```

Comment:
adding this as it will then add test results in cypress  

(currently spits out a warning when running cypress as its registered as reporter in `cypress-reporter-config.js`)